tokens/figma/utils/api-request.mjs

```diff
@@ -0,0 +1,50 @@
+import fetch from "node-fetch";
+
+async function getFigmaData(
+  FILE_KEY,
+  FIGMA_TOKEN
+) {
+  const response = await fetch(
+    `https://api.figma.com/v1/files/${FILE_KEY}/variables/local`,
+    {
+      method: "GET",
+      headers: {
+        "X-Figma-Token": FIGMA_TOKEN,
+        "Content-Type": "application/json",
+      },
+    }
+  );
+  if (!response.ok) {
+    throw new Error(
+      `Error fetching Figma data: ${response.statusText}`
+    );
+  }
+  return response.json();
```

Comment:
response.json() is an async function. Consider waiting for the result

Reply:
Fixed